<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdd</class>
 <widget class="QDialog" name="DialogAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hinzuf√ºgen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Geburtsdatum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Lieblings-Genre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="genreEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
